LWRP.md

```diff
@@ -1370,9 +1370,9 @@ LWRP `apilistener` creates an icinga `ApiListener` object.
 
 ```ruby
 icinga2_apilistener 'master' do
-  cert_path 'ConfigDir + "/icinga2/pki/" + NodeName + ".crt"'
-  key_path 'ConfigDir + "/icinga2/pki/" + NodeName + ".key"'
-  ca_path 'ConfigDir + "/icinga2/pki/ca.crt"'
+  cert_path 'ConfigDir + "/pki/" + NodeName + ".crt"'
+  key_path 'ConfigDir + "/pki/" + NodeName + ".key"'
+  ca_path 'ConfigDir + "/pki/ca.crt"'
```

Comment:
These paths have been deprecated in 2.8. While at it, think about a possible change here to `/var/lib...` as well.

Reply:
https://github.com/Icinga/icinga2/blob/master/doc/16-upgrading-icinga-2.md#upgrading-to-v28- oh shoot.

> Icinga 2 automatically migrates the certificates to the new default location if they are configured and detected in /etc/icinga2/pki.

makes it sound like Icinga is papering over that under the covers, at least. I kind of want to address that in a separate PR.

Reply:
It has been sitting there, but likely not forever. Just wanted to mention it, thanks for taking care :)